pallets/subtensor/src/certificate.rs

```diff
@@ -0,0 +1,4 @@
+use sp_std::vec::Vec;
+
+// TODO Certificate Encryption Pubkey
+pub type Certificate = Vec<u8>;
```

Comment:
As odd pointed out in slack, having `[u8; 32]` would be more appropriate than `Vec<u8>` as the certificates are supposed to be 256-bit in length. We don't want to allow users to store data of arbitrary size here.